Plugin~/WebRTCPlugin/GraphicsDevice/IGraphicsDevice.h

```diff
@@ -15,11 +15,13 @@ class IGraphicsDevice {
     virtual bool InitV() = 0;
     virtual void ShutdownV() = 0;
     virtual ITexture2D* CreateDefaultTextureV(uint32_t width, uint32_t height) = 0;
+    virtual ITexture2D* CreateStagingTextureV(uint32_t width, uint32_t height) { return NULL; }
```

Comment:
returning NULL here defeats the purpose of making an interface class.
Rather, please write `return null` in each derived class and put assert or log so that it is clear that the implementation for that graphics device is not yet implemented.